packages/http-client-csharp/generator/Microsoft.Generator.CSharp.Input/src/InputTypes/InputOperationKinds.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+using System;
+using System.ComponentModel;
+
+namespace Microsoft.Generator.CSharp.Input
+{
+    /// <summary>
+    /// The kind of operation.
+    /// </summary>
+    public readonly partial struct InputOperationKinds : IEquatable<InputOperationKinds>
```

Comment:
I think rather having an extensible enum, could we just make this as a static class containing string values? It could be much simpler if we want to switch on things.
Or maybe we could just make this as a regular enum

Reply:
it has to be public for people who override the get method need to know what "kind" it is.  I think this should go onto the InputOperation object directly  maybe?